src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -80,11 +81,26 @@ public virtual void SetupChannel(IEnumerable<string> sources)
 
                 var portNumber = this.RequestSender.InitializeCommunication();
                 var processId = this.processHelper.GetCurrentProcessId();
-                var connectionInfo = new TestRunnerConnectionInfo { Port = portNumber, RunnerProcessId = processId, LogFile = this.GetTimestampedLogFile(EqtTrace.LogFile) };
+
+                var logFileName = this.GetTimestampedLogFile(EqtTrace.LogFile);
```

Comment:
You are right but there is one scenario where consumer of EqtTrace has to check for permission.
1) Let say our command to run test is "vstest.console.exe  test.dll /diag:logfile.txt"
2) Suppose we are running test from "c:\Program File" and this location already has a file with name logfile.txt.

In this case vstest will be able open and update logfile.txt but when testhost will try to create logfile.host.*.txt file it will fail with unauthorized exception.